cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -716,7 +716,7 @@ Slice::CsGenerator::writeMarshalUnmarshalCode(
     {
         if (marshal)
         {
-            if (!isValueType(st))
```

Comment:
isMappedToClass is basically an alias that I find more readable than `!isValueType(st)`